tests/features/repo.feature

```diff
@@ -50,6 +50,19 @@ Feature: Subcommand: dab repo
 		And the output should match /^dotfiles3\s*|\s*not downloaded/
 		And the output should match /^dotfiles4\s*|\s*downloaded/
 
+	Scenario: Can filter listed repositories
+		Given a file named "~/.config/dab/repo/dotfiles3/url" with:
+		"""
+		https://github.com/Nekroze/dotfiles.git
+		"""
+		And the directory "~/dab/dotfiles3/.git/" should not exist
+		And I successfully run `dab repo add dotfiles4 https://github.com/Nekroze/dotfiles.git`
```

Comment:
To avoid issues between test reruns and due to the order being randomized each run, each of these repo tests need a unique repo not used in another test, `dotfiles3` and `dotfiles4` are already used and may cause flakey tests if reused. Instead you can move this to the end and increment the numbers or generate a random number/id to use.